src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -171,13 +170,17 @@ func useGoodOldConfig(configDir, configPath string) bool {
 // Init initializes the configuration system.
 func Init(version string) (*Config, error) {
 	file := "lantern-" + version + ".yaml"
-	configDir, configPath, err := InConfigDir(file)
+	_, configPath, err := InConfigDir(file)
 	if err != nil {
 		log.Errorf("Could not get config path? %v", err)
 		return nil, err
 	}
-	ok := useGoodOldConfig(configDir, configPath)
-	if !ok {
+	run := isGoodConfig(configPath)
```

Comment:
For now we go back to just using configs for this version, as the old configs will typically have bad CloudFlare masquerades we can't use and that will cause problems.